app/containers/EditorPage.tsx

```diff
@@ -12,7 +16,8 @@ function mapDispatchToProps(dispatch: Dispatch) {
   return bindActionCreators(
     {
       changeAvro: changeAvroWithDispatch,
-      avroMouseMove
+      avroMouseMove,
+      changeAvroPristine: changeAvroPristineWithDispatch
```

Comment:
I am not sure we need a `changeAcroPristine` action. I think the `changeAvro` could set the `isAvroPristine` state to `true` and the `saveAvroAction` could set it fo `false`